hostscripts/gatehost/freshadminvm

```diff
@@ -25,13 +25,15 @@ common_scripts="mkcloud-onhost.sh mkcloud-common.sh mkcloud-driver-$mkclouddrive
 for script in $common_scripts; do
     source ${scripts_lib_dir}/$script
 done
-admin_node_disk=/dev/system/$n
-[ -e $admin_node_disk ] || admin_node_disk=/dev/gate/$n
-[ -e $admin_node_disk ] || exit 51
+if [ -z "$admin_node_disk" ]; then
+        admin_node_disk=/dev/system/$n
+        [ -e $admin_node_disk ] || admin_node_disk=/dev/gate/$n
+        [ -e $admin_node_disk ] || exit 51
```

Comment:
Thats still insecure: no quotes in single brackets.
```
+ [[ -e $admin_node_disk ]] || admin_node_disk=/dev/gate/$n
+ [[ -e $admin_node_disk ]] || exit 51
```

Reply:
$admin_node_diskis already checked for non-emptiness.

Reply:
I would also prefer the `[[` style here. It's more consistent.